client/src/context/AppContext.js

```diff
@@ -1,4 +1,14 @@
-import { createContext } from 'react';
+import { createContext, useContext } from 'react';
 
 const AppContext = createContext();
-export default AppContext;
+export { AppContext };
+
+export function useAlert() {
+  const { alert, setAlert } = useContext(AppContext);
+  return { alert, setAlert };
+}
+
+export function useUser() {
+  const { userId } = useContext(AppContext);
+  return { userId };
```

Comment:
I would have written this so that `useUser` simply returns the userId instead of an object. If you had an actual `user` object you could return that would be ideal, and maybe that will happen as things evolve.